civis/tests/test_io.py

```diff
@@ -419,3 +433,24 @@ def test_split_schema_tablename_raises():
     s = "table.with.too.many.periods"
     with pytest.raises(ValueError):
         civis.io._tables.split_schema_tablename(s)
+
+
+@mock.patch.object(civis.io._tables, '_sql_script', autospec=True,
+                   return_value=[-1000, 1000])
```

Comment:
I've done things like this myself in the past, but I've come to think that when you're mocking, it's best to use inputs and outputs which are valid for the function being mocked. The job ID and run ID are always going to be positive integers, so maybe you could use something like `return_value=[700, 1000]`?